src/app/index.html

```diff
@@ -4,27 +4,45 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>__TITLE__</title>
-    {% if (o.htmlWebpackPlugin.options.bundle) { %}
-      {% if (o.htmlWebpackPlugin.files && o.htmlWebpackPlugin.files.chunks) { %}
-        {% if (o.htmlWebpackPlugin.files.chunks.main && o.htmlWebpackPlugin.files.chunks.main.css) { %}
-          {% for (var chunk in o.htmlWebpackPlugin.files.chunks.main.css) { %}
-            <link rel="stylesheet" href="{%=o.htmlWebpackPlugin.files.chunks.main.css[chunk]%}" media="all">
-          {% } %}
-        {% } %}
-      {% } %}
-    {% } %}
+    <% for (var chunk in htmlWebpackPlugin.files.chunks.main.css) { %>
```

Comment:
I think you lost `<title>` here. Minor, but it could be easily added, see e.g. https://github.omnilab.our.buildo.io/buildo/aliniq/blob/master/qia/web/src/app/index.html#L16